src/style.css

```diff
@@ -1,3 +1,90 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --main-color: #00ACDC;
+}
+
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
+  padding: 0;
+}
+
+.header {
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo {
+  display: inline-block;
+  text-decoration: none;
+  margin: 0;
+  padding: 0;
+  width: 40px;
+  height: 40px;
+}
+
+.box-item {
+  margin: 20px;
+  display: inline-block;
+  list-style: none;
+}
+
+.box-link {
+  color: #000;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: normal;
+  text-transform: uppercase;
+  text-decoration: none;
+  cursor: pointer;
+  position: relative;
+}
+
+.box-item.is-active .box-link,
```

Comment:
you don't need a combinator selector here, because `is-active` should be on the link
![image](https://github.com/mate-academy/layout_moyo-header/assets/86923089/851c379c-c3f6-4065-80af-4f09ab9da99d)